src/api_authentication_app_skeleton/spec/requests/api/sign_ups_spec.cr

```diff
@@ -0,0 +1,20 @@
+require "../../spec_helper"
+
+describe "SignUps" do
+  it "creates user on sign up" do
+    request = AppRequest.new
+
+    request.post("/api/sign_ups", ({
+      "sign_up:email"                 => "test@email.com",
+      "sign_up:password"              => "password",
+      "sign_up:password_confirmation" => "password",
+    }))
+
+    request.response.status_code.should eq 200
+
+    new_user = UserQuery.new.email("test@email.com").first
+    new_user.should_not be_nil
+
```

Comment:
I think these lines 14 and 17 can be removed. You may also want move it above line 13 so the expectations are all in a row